src/main/java/codeu/model/data/User.java

```diff
@@ -58,4 +75,9 @@ public String getPassword() {
   public Instant getCreationTime() {
     return creation;
   }
+
+  /** Returns the about me of this User. */
+  public String getAboutMe() {
```

Comment:
Will you need a setAboutMe as well, since it could be changed by editing the profile page?